### PR TITLE
Put YouTube channel link on About page

### PR DESCRIPTION
**Pull request by KeithYokoma**

### CODE REVIEW

**Comment by konifar** [comment]
Good!
And if you want to add YouTube link by each sessions, you can use `movieUrl` value in Session model from `res/raw/sessions_xx.json`.

https://github.com/konifar/droidkaigi2016/blob/master/app/src/main/java/io/github/droidkaigi/confsched/model/Session.java#L72

**Comment by KeithYokoma** [comment]
Nice! I think URLs for each sessions is not available until videos uploaded, so for now this PR is ready to go!

File: app/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -72,6 +72,7 @@
     <string name="about_social">اجتماعي</string>
     <string name="about_terms">شروط الاستخدام</string>
     <string name="about_license">رخصة</string>
+    <string name="about_youtube">YouTube قناة</string>
```
Comment (konifar) [inline app/src/main/res/values-ar/strings.xml]: Could you change to `&#8207;YouTube قناة` ?

When the first character is NOT Arabic, RTL is ignored. `&#8207;`, called Left-To-Right Mark, force text RTL.

**Comment by hkurokawa** [comment]
@KeithYokoma Thank you. I should do that. :bow: 

By the way, the video URL is not fixed until it is uploaded to YouTube so I guess most of the video URLs are updated only after the event. Just for your information.

**Comment by KeithYokoma** [comment]
Fixed RTL string!

**Pull request merged**